src/NodeRegistry.sol

```diff
@@ -240,70 +210,38 @@ contract NodeRegistry is INodeRegistry, AccessControlDefaultAdminRules, ERC721 {
 
-    /// @inheritdoc INodeRegistry
+    /**
+     * @inheritdoc INodeRegistry
+     */
     function getAllNodesCount() public view returns (uint256 nodeCount) {
         return _nodeCounter;
     }
 
-    /// @inheritdoc INodeRegistry
+    /**
+     * @inheritdoc INodeRegistry
+     */
     function getNode(uint256 nodeId) public view returns (Node memory node) {
         _revertIfNodeDoesNotExist(nodeId);
         return _nodes[nodeId];
     }
 
-    /// @inheritdoc INodeRegistry
-    function getActiveApiNodes() external view returns (NodeWithId[] memory activeNodes) {
-        activeNodes = new NodeWithId[](_activeApiNodes.length());
-
-        for (uint32 i; i < _activeApiNodes.length(); ++i) {
-            uint256 nodeId = _activeApiNodes.at(i);
-
-            activeNodes[i] = NodeWithId({ nodeId: nodeId, node: _nodes[nodeId] });
-        }
-    }
-
-    /// @inheritdoc INodeRegistry
-    function getActiveApiNodesIDs() external view returns (uint256[] memory activeNodesIDs) {
-        return _activeApiNodes.values();
-    }
-
-    /// @inheritdoc INodeRegistry
-    function getActiveApiNodesCount() external view returns (uint256 activeNodesCount) {
-        return _activeApiNodes.length();
-    }
-
-    /// @inheritdoc INodeRegistry
-    function getApiNodeIsActive(uint256 nodeId) external view returns (bool isActive) {
-        return _activeApiNodes.contains(nodeId);
-    }
-
-    /// @inheritdoc INodeRegistry
-    function getActiveReplicationNodes() external view returns (NodeWithId[] memory activeNodes) {
-        activeNodes = new NodeWithId[](_activeReplicationNodes.length());
-
-        for (uint32 i; i < _activeReplicationNodes.length(); ++i) {
-            uint256 nodeId = _activeReplicationNodes.at(i);
-
-            activeNodes[i] = NodeWithId({ nodeId: nodeId, node: _nodes[nodeId] });
-        }
-    }
-
-    /// @inheritdoc INodeRegistry
-    function getActiveReplicationNodesIDs() external view returns (uint256[] memory activeNodesIDs) {
-        return _activeReplicationNodes.values();
-    }
-
-    /// @inheritdoc INodeRegistry
-    function getActiveReplicationNodesCount() external view returns (uint256 activeNodesCount) {
-        return _activeReplicationNodes.length();
+    /**
+     * @inheritdoc INodeRegistry
+     */
+    function getIsCanonicalNode(uint256 nodeId) external view returns (bool isCanonicalNode) {
+        return _canonicalNetworkNodes.contains(nodeId);
     }
 
-    /// @inheritdoc INodeRegistry
-    function getReplicationNodeIsActive(uint256 nodeId) external view returns (bool isActive) {
-        return _activeReplicationNodes.contains(nodeId);
+    /**
+     * @inheritdoc INodeRegistry
+     */
+    function getMaxActiveNodes() external view returns (uint8 maxNodes) {
+        return _maxActiveNodes;
     }
 
-    /// @inheritdoc INodeRegistry
+    /**
+     * @inheritdoc INodeRegistry
+     */
     function getNodeOperatorCommissionPercent() external view returns (uint256 commissionPercent) {
```

Comment:
This fucntion could be removed entirely since `nodeOperatorCommissionPercent` is, and should remain, public.